### DISCUSSION TITLE
1.7.14 ready to go

### DISCUSSION THREAD

**Discussion by gbrail** [mozilla/rhino#1145] [Announcements]
Thanks for all of your help and patience -- I merged the last few things into the master branch, and some time tomorrow I'll publish 1.7.14. Then we can keep moving forward!

**Comment by p-bakker** [comment]
Great work @gbrail (and all others who contributed!)

**Comment by p-bakker** [comment]
What to do with #1107? It's currently linked to the v1.7.14 milestone, but it's not really related to Rhino (code)
